skyllh/core/analysis_utils.py

```diff
@@ -157,8 +157,8 @@ def calculate_critical_ts_from_gamma(
     r = minimize(obj, x0, bounds=bounds)
     pars = r.x
     
-    interval = gamma.interval(1 - h0_ts_quantile, a=pars[0], scale=pars[1])
-    critical_ts = interval[1]
+    norm = alpha * (alpha_prime / (1-gamma.cdf(eta, a=pars[0], scale=pars[1]))
+    critical_ts = gamma.ppf(1 - 1./norm*h0_ts_quantile, a=pars[0], scale=pars[1])
```

Comment:
I think this only works for critical_ts > eta. 
We should think about checking/ensuring that his holds true.